container-images/scripts/build_rag.sh

```diff
@@ -30,7 +30,7 @@ $2"
 
 packages="git-core gcc gcc-c++"
 if [ "${GPU}" = "cuda" ]; then
-    packages+=" libcudnn9-devel-cuda-12 libcusparselt0 cuda-cupti-12\*"
```

Comment:
You can drop all this if you want, I caught this in the other PR, when we want to do a list of args, arrays are safer, less error prone to globbing type errors. It shuts up all the linters, etc. mentioning this also.